index.js

```diff
@@ -248,9 +248,9 @@ function provider (registry, { Biome, version }) {
       if (str.startsWith('minecraft:')) str = str.substring(10)
       const name = str.split('[', 1)[0]
       const propertiesStr = str.slice(name.length + 1, -1).split(',')
-      if (version.type === 'pc') {
+      if (!str.includes('["')) {
```

Comment:
can you add a comment explaining why the presence of [" should determine one or the other branch? it's pretty unclear from the context

Reply:
To be entirely honest I just did what was asked of me in that comment

Reply:
You should add a comment to explain the check on syntax, as there are two possible syntax for the block string